docs/architecture/011-capi-elements-as-components.md

```diff
@@ -0,0 +1,41 @@
+# Elements as Components
+
+## Context
+
+CAPI provides a representation of content which is strongly typed and like components.
+
+## Decision
+
+### CAPI Elements as Components
```

Comment:
Sorry to be the subheading police 🚓 Can we just keep the following subheadings:

- unaltered elements
- altered elements

Maybe we can move the "Functionality" section above these subheadings